azurerm/internal/services/cdn/parse/cdn_endpoint.go

```diff
@@ -36,3 +44,8 @@ func CdnEndpointID(input string) (*CdnEndpointId, error) {
 
 	return &endpoint, nil
 }
+
+func (id CdnEndpointId) ID(subscription string) string {
```

Comment:
_minor_ so this is consistent with the other formatters can we make this:

```suggestion
func (id CdnEndpointId) ID(subscriptionId string) string {
```